Assets/Scripts/Wizzard/WizzardController.cs

```diff
@@ -1,24 +1,25 @@
-﻿using System;
+﻿using Assets.Scripts.Spells;
+using System;
+using System.Collections.Generic;
 using UnityEngine;
 
 public class WizzardController : MonoBehaviour
 {
-    [HideInInspector]
-    public bool facingRight = true;
-    public float moveForce = 30f;
-    public float maxSpeed = 1f;
-    public GameObject FireballBallistic;
-    public GameObject FireballStraight;
-    public GameObject FireballTime;
+    [HideInInspector] public bool facingRight = true;
     [HideInInspector] public bool InputEnabled;
     [HideInInspector] public int Angle;
-    private int FireStrength = 500;
-
+    public float moveForce = 30f;
+    public float maxSpeed = 1f;
+    public int FireStrength = 500;
+    
+    private int currentSpell;
```

Comment:
Jest to gdzies ustawiane?

Reply:
Domyślnie int przyjmuje wartość 0 - także będzie ustawiony pierwszy czar z listy template :).